planning-release.md

```diff
@@ -0,0 +1,39 @@
+# Planning a release on XebiaLabs
+
+With XebiaLabs's XL Release, you can plan, automate, and optimize your pipeline visually in a neat, step-by-step diagram which is referred to as a "release". 
```

Comment:
I think we are miss-understanding from the previous comment. 

```
2. Provide some brief information about "XebiaLabs-Appium-ReactNative", e.g What will this Repo show to the end user.
```

I mean that you should provide some brief information about "XebiaLabs-Appium-ReactNative" in README.md and you have done it in the newest commit